src/styles/mixins/_forms.scss

```diff
@@ -219,3 +224,198 @@
     margin-right: 0;
   }
 }
+
+@mixin fd-form-input() {
```

Comment:
It's the same we have above, at the top of the file, in `fd-form-states` mixin. Can we remove one of these and left only one?